ecl2df/gruptree.py

```diff
@@ -225,10 +223,44 @@ def edge_dataframe2dict(dframe):
 
     children, parents = zip(*edges)
     roots = set(parents).difference(children)
-    trees = []
-    for root in list(roots):
-        trees.append({root: subtrees[root] for root in roots})
-    return trees
+    return [{root: subtrees[root]} for root in sorted(roots)]
+
+
+def _sub_tree_from_dict(nested_dict, name):
+    """Internal recursive function for generating trees."""
```

Comment:
Inconsistent use of period at end of docstring.

Reply:
Yes, PEP-257 says we should end docstring sentences with a period. So it is the other functions that need a cleanup then.

Private (hidden) functions starting with a `_` are a blessing when you later want to change the API.. :)